src/components/game/graphics/objects/piece/UpdatePiece.js

```diff
@@ -0,0 +1,12 @@
+import { drawCell } from "../cell/DrawCell";
+import { updateCell } from "../cell/UpdateCell";
+
+export const updatePiece = (piece) => {
+    // Fill in cells from the coreState current piece.
+    var [x, y] = [0, 0];
```

Comment:
Clean up useless code/comments; updatePiece is totally okay as just a wrapper right now